lib/dmv_facilities.rb

```diff
@@ -0,0 +1,30 @@
+class DMVFacility
```

Comment:
I think it might make more sense to call this class `FacilityFactory` to match the Vehicle Factory naming convention